novem/vis/custom.py

```diff
@@ -0,0 +1,37 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from novem.vis.plot import Plot
+
+
+class NovemCustom:
+    def __init__(self, api: "Plot") -> None:
+        self.api: "Plot" = api
+
```

Comment:
I think this is a nice style:
```python
from dataclass import dataclass

@dataclass(frozen=True)
class NovemCustom:
    api: "Plot"

   <functions go here>
```
(not tested)

Reply:
Nice, incorporated